doc/sphinx/changes.rst

```diff
@@ -8,6 +8,519 @@ Recent changes
+  Ltac `match` does not fail when the term to match contains an unfolded primitive projection
+  (`#15559 <https://github.com/coq/coq/pull/15559>`_,
+  fixes `#15554 <https://github.com/coq/coq/issues/15554>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  ``Ltac2`` understands :n:`toplevel_selector` and obeys :opt:`Default Goal Selector`.
+  Note that ``par:`` is buggy when combined with :tacn:`abstract`. Unlike ``Ltac1`` even ``par: abstract tac`` is not properly treated.
+  (`#15378 <https://github.com/coq/coq/pull/15378>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  Ltac2 `Int` functions `div`, `mod`, `asr`, `lsl`, `lsr`, `land`, `lor` , `lxor` and `lnot`.
+  (`#15637 <https://github.com/coq/coq/pull/15637>`_,
+  by Michael Soegtrop).
+- **Fixed:**
+  Ltac2 `apply` and `eapply` not unifying with implicit arguments;
+  unification inconsistent with `exact` and `eexact`
+  (`#15741 <https://github.com/coq/coq/pull/15741>`_,
+  by Ramkumar Ramachandra).
+
+SSReflect
+^^^^^^^^^
+
+- **Fixed:**
+  :tacn:`have`, :tacn:`suff` and :tacn:`wlog` support goals in `sProp`
+  (`#15121 <https://github.com/coq/coq/pull/15121>`_,
+  by Enrico Tassi).
+
+Commands and options
+^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  :cmd:`Module` now only allows parentheses around module arguments. For instance, ``Module M := (F X).`` is now a parsing error
+  (`#15355 <https://github.com/coq/coq/pull/15355>`_,
+  by Gaëtan Gilbert).
+- **Changed:**
+  :cmd:`Fail` no longer catches anomalies, which it has done since Coq version 8.11.
+  Now it only catches user errors
+  (`#15366 <https://github.com/coq/coq/pull/15366>`_,
+  by Hugo Herbelin).
+- **Changed:**
+  :ref:`program_definition` in universe monomorphic mode does not accept non-extensible universe declarations
+  (`#15424 <https://github.com/coq/coq/pull/15424>`_,
+  fixes `#15410 <https://github.com/coq/coq/issues/15410>`_,
+  by Gaëtan Gilbert).
+- **Changed:**
+  The algorithm for name generation of anonymous variables
+  for ``Program`` subproofs is now the same as the one
+  used in the general case. This can create incompatibilities
+  in scripts relying on such autogenerated names. The old
+  scheme can be reactivated using the deprecated flag
+  :flag:`Program Naming`
+  (`#15442 <https://github.com/coq/coq/pull/15442>`_,
+  by Pierre-Marie Pédrot).
+- **Removed:**
+  `Universal Lemma Under Conjunction` flag, that was deprecated in 8.15
+  (`#15268 <https://github.com/coq/coq/pull/15268>`_,
+  by Théo Zimmermann).
+- **Removed:**
+  :cmd:`Abort` no longer takes an :n:`@ident` as an argument (it has been ignored since 8.5)
+  (`#15669 <https://github.com/coq/coq/pull/15669>`_,
+  by Gaëtan Gilbert).
+- **Removed:**
+  `Simplex` flag, that was deprecated in 8.14.
+  :tacn:`lia` and :tacn:`lra` will always use the simplex solver (that was already the default behaviour).
+  (`#15690 <https://github.com/coq/coq/pull/15690>`_,
+  by Frédéric Besson).
+- **Deprecated:**
+  :cmd:`Add LoadPath` and :cmd:`Add Rec LoadPath`
```

Comment:
We should probably repeat here the message that we have put elsewhere in the doc:

>If this command is an important feature for you, please open an issue at https://github.com/coq/coq/issues and explain your workflow